index.js

```diff
@@ -57,27 +56,15 @@ const main = async () => {
   if (process.argv.slice(2).length > 0) {
     // Run a single mine
     const name = process.argv.slice(2)[0];
-
-    try {
-      const mine = require(`./mines/${name}.js`).default;
-
-      totalScore += await runMine(mine, name);
-    } catch (error) {
-      console.error("Invalid mine name");
-    }
```

Comment:
This was catching other errors and not allowing them to print out. Instead we can let the top level catch the error and log it.